account.go

```diff
@@ -222,92 +239,62 @@ func (ea *ExternalAccount) UnmarshalJSON(b []byte) error {
-	PersonalAddressKana   Address              `json:"personal_address_kana"`
-	PersonalAddressKanji  Address              `json:"personal_address_kanji"`
-	PhoneNumber           string               `json:"phone_number"`
-	DOB                   DOB                  `json:"dob"`
-	Verification          IdentityVerification `json:"verification"`
-	SSN                   string               `json:"ssn_last_4"`
-	SSNProvided           bool                 `json:"ssn_last_4_provided"`
-	PersonalID            string               `json:"personal_id_number"`
-	PersonalIDProvided    bool                 `json:"personal_id_number_provided"`
-	BusinessTaxID         string               `json:"business_tax_id"`
-	BusinessTaxIDProvided bool                 `json:"business_tax_id_provided"`
-	BusinessVatID         string               `json:"business_vat_id"`
+	AdditionalOwnersEmpty bool `form:"additional_owners,empty"`
+
+	BusinessName          string               `json:"business_name" form:"business_name"`
+	BusinessNameKana      string               `json:"business_name_kana" form:"business_name_kana"`
+	BusinessNameKanji     string               `json:"business_name_kanji" form:"business_name_kanji"`
+	Address               Address              `json:"address" form:"address"`
+	AddressKana           Address              `json:"address_kana" form:"address_kana"`
+	AddressKanji          Address              `json:"address_kanji" form:"address_kanji"`
+	First                 string               `json:"first_name" form:"first_name"`
+	FirstKana             string               `json:"first_name_kana" form:"first_name_kana"`
+	FirstKanji            string               `json:"first_name_kanji" form:"first_name_kanji"`
+	Gender                Gender               `json:"gender" form:"gender"`
+	Last                  string               `json:"last_name" form:"last_name"`
+	LastKana              string               `json:"last_name_kana" form:"last_name_kana"`
+	LastKanji             string               `json:"last_name_kanji" form:"last_name_kanji"`
+	MaidenName            string               `json:"maiden_name" form:"maiden_name"`
+	PersonalAddress       Address              `json:"personal_address" form:"personal_address"`
+	PersonalAddressKana   Address              `json:"personal_address_kana" form:"personal_address_kana"`
+	PersonalAddressKanji  Address              `json:"personal_address_kanji" form:"personal_address_kanji"`
+	PhoneNumber           string               `json:"phone_number" form:"phone_number"`
+	DOB                   DOB                  `json:"dob" form:"dob"`
+	Verification          IdentityVerification `json:"verification" form:"verification"`
+	SSN                   string               `json:"ssn_last_4" form:"ssn_last_4"`
+	SSNProvided           bool                 `json:"ssn_last_4_provided" form:"-"` // Not updatable
+	PersonalID            string               `json:"personal_id_number" form:"personal_id_number"`
```

Comment:
so this field does not exist in the response, it only exists as a parameter. I guess the original implementation was wrong. No big deal but was curious how it would work for a case like this so wanted to flag

Reply:
Good catch. It turns out that there were quite a few errors in the original implementation.

I just patched all this in 20a4f6c which stops trying to decode fields that will never be there in a resource (only their matching "provided" fields should be expected in JSON), and only encodes them as form parameters. I think it's correct now.